src/System.Xml.XDocument/ref/System.Xml.XDocument.cs

```diff
@@ -211,6 +212,7 @@ public partial class XDocumentType : System.Xml.Linq.XNode
         public override System.Threading.Tasks.Task WriteToAsync(System.Xml.XmlWriter writer, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
     [System.Xml.Serialization.XmlSchemaProvider(null, IsAny=true)]
+    [System.ComponentModel.TypeDescriptionProvider("MS.Internal.Xml.Linq.ComponentModel.XTypeDescriptionProvider`1[[System.Xml.Linq.XElement, System.Xml.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]],System.ComponentModel.TypeConverter")]
```

Comment:
on github it looks like there is something wrong with indentation in here (this whole file) - please make sure this is correct

Reply:
This file is no longer part of this commit.  I decided not to expose this in the reference assembly.